mock-core-configs/etc/mock/templates/centos-stream-9.tpl

```diff
@@ -0,0 +1,48 @@
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['dist'] = 'el9'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '9'
+config_opts['package_manager'] = 'dnf'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['bootstrap_image'] = 'quay.io/centos/centos:stream9'
+
+config_opts['dnf_vars'] = { 'stream': '9-stream',
+                            'contentdir': 'centos',
+                          }
+
+config_opts['dnf.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+mdpolicy=group:primary
+best=1
+protected_packages=
+module_platform_id=platform:el8
+user_agent={{ user_agent }}
+
+[baseos]
+name=CentOS Stream $releasever - BaseOS
+baseurl=https://composes.stream.centos.org/test/latest-CentOS-Stream/compose/BaseOS/$basearch/os/
```

Comment:
**If** we move forward with this before the content is mirrored, we shouldn't use the `test` composes, we should use the `production` composes, e.g. `https://composes.stream.centos.org/production/latest-CentOS-Stream/compose/BaseOS/$basearch/os/`.

Reply:
Good point, thanks.  @carlwgeorge   Can you guess the time when mirrors will be working?  If that is more than a week or two, we'd rather release these now, and release again with the mirrorlist variant once  available.

Reply:
I don't have an exact guess, but it is safe to say it will take longer than two weeks.